junction/base/utils.py

```diff
@@ -3,13 +3,13 @@
 
 def get_date_diff_display(start, end):
     if end.year != start.year:
-        return "%s - %s" % (start.strftime("%-d %b %Y"), end.strftime("%-d %b %Y"))
+        return "%s - %s" % (start.strftime("%d %b %Y"), end.strftime("%d %b %Y"))
```

Comment:
Can we use f-strings here, please?

Reply:
yes, we can. 
![image](https://user-images.githubusercontent.com/69859505/211567336-a81d7151-b1ec-40ee-b177-6d79bcb2e55d.png)

changes done!